apps/examples/led_test/Kconfig

```diff
@@ -0,0 +1,27 @@
+#
+#For a description of the syntax of this configuration file,
+#see kconfig-language at https:	//www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_LED
+	bool "LED example"
+	default n
+	---help---
+		Enable LED Example
+
+if EXAMPLES_LED
+
+if ARCH_BOARD_RTL8730E
```

Comment:
Let's remove this. If we want to add this, then we should locate this in `apps/example/board_specific`.
Just add `config LED_PIN` only. but, we should distinguish the config for app and driver. I think we should add `EXAMPLES_` prefix, I mean `config EXAMPLES_LED_PIN` is better.